### PR TITLE
[HOTFIX] Add checkpoint() method implementation

### PR DESCRIPTION
**Pull request by Leemoonsoo**
### What is this PR for?
While merging https://github.com/apache/incubator-zeppelin/pull/697, NotebookRepo API changed.
And it breaks master branch. This PR implement checkpoint() method to fix it.

### What type of PR is it?
Hot Fix

### Todos
* [x] - Implement checkpoint()

### Is there a relevant Jira issue?
ZEPPELIN-656

### How should this be tested?
build master branch

### Screenshots (if appropriate)
N/A

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by khalidhuseynov** [comment]
Thanks for catching up with latest API, looks good!

File: zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/AzureNotebookRepo.java [hunk 1]
```diff
@@ -204,4 +204,10 @@ public void remove(String noteId) throws IOException {
   @Override
   public void close() {
   }
+
+  @Override
+  public void checkpoint(String noteId, String checkPointName) throws IOException {
+    // no-op
+    LOG.info("Checkpoint feature isn't suported in {}", this.getClass().toString());
```
Comment (felixcheung) [inline zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/AzureNotebookRepo.java]: suported -> supported

**Comment by felixcheung** [comment]
looks good

**Comment by granturing** [comment]
looks good, thanks for handling that

**Comment by Leemoonsoo** [comment]
Thanks guys for review.
I'm merging it as a hotfix

**Pull request closed without merging**